package.json

```diff
@@ -96,7 +96,6 @@
     "ts-node": "^10.3.0",
     "tsconfig": "^7.0.0",
     "tsconfig-paths": "^3.11.0",
-    "tslib": "^2.3.1",
```

Comment:
@apolkingg8 I see you removed the newer version, is there a reason we need `2.0.0` instead of `2.3.1`? It's nice to stick with the latest one since it often includes performance improvements.

Reply:
Ah, sorry I didn't notice the version. Will fix it in the next PR.

Reply:
Thanks 👍